examples/mwaa/main.tf

```diff
@@ -0,0 +1,299 @@
+    dag_processing_logs = {
+      enabled   = true
+      log_level = "INFO"
+    }
+
+    scheduler_logs = {
+      enabled   = true
+      log_level = "INFO"
+    }
+
+    task_logs = {
+      enabled   = true
+      log_level = "INFO"
+    }
+
+    webserver_logs = {
+      enabled   = true
+      log_level = "INFO"
+    }
+
+    worker_logs = {
+      enabled   = true
+      log_level = "INFO"
+    }
+  }
+
+}
+
+#------------------------------------------------------------------------
+# AWS VPC Module
+#------------------------------------------------------------------------
+module "aws_vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
+
+  name = local.name
+  cidr = local.vpc_cidr
+  azs  = local.azs
+
+  public_subnets  = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k)]
+  private_subnets = [for k, v in local.azs : cidrsubnet(local.vpc_cidr, 8, k + 10)]
+
+  enable_nat_gateway   = true
+  create_igw           = true
+  enable_dns_hostnames = true
+  single_nat_gateway   = true
+
+  public_subnet_tags = {
+    "kubernetes.io/cluster/${local.name}" = "shared"
+    "kubernetes.io/role/elb"              = "1"
+  }
+
+  private_subnet_tags = {
+    "kubernetes.io/cluster/${local.name}" = "shared"
+    "kubernetes.io/role/internal-elb"     = "1"
+  }
+}
+
+#------------------------------------------------------------------------
+# AWS EKS Blueprints Module
+#------------------------------------------------------------------------
+module "eks_blueprints" {
+  source = "../.."
+
+  cluster_name       = local.name
+  cluster_version    = var.eks_cluster_version
+  vpc_id             = module.aws_vpc.vpc_id
+  private_subnet_ids = module.aws_vpc.private_subnets
+
+  # Attach additional security group ids to Worker Security group ID
+  worker_additional_security_group_ids = [] # Optional
+
+
+  # Add MWAA IAM Role to aws-auth configmap
+  map_roles = [
+    {
+      rolearn  = "${module.mwaa.mwaa_role_arn}" # The ARN of the IAM role
+      username = "mwaa-role"                    # The user name within Kubernetes to map to the IAM role
+      groups   = ["system:masters"]             # A list of groups within Kubernetes to which the role is mapped; Checkout K8s Role and Rolebindings
+    }
+  ]
+
+  # EKS MANAGED NODE GROUPS
+  managed_node_groups = {
+    # Managed Node groups with minimum config
+    mg5 = {
+      node_group_name = "mg5"
+      instance_types  = ["m5.large"]
+      min_size        = "2"
+      disk_size       = 100 # Disk size is used only with Managed Node Groups without Launch Templates
+    }
+  }
+}
+
+#------------------------------------------------------------------------
+# Kubernetes Add-on Module
+#------------------------------------------------------------------------
+module "eks_blueprints_kubernetes_addons" {
+  source = "../../modules/kubernetes-addons"
+
+  eks_cluster_id = module.eks_blueprints.eks_cluster_id
+
+  enable_metrics_server     = false
```

Comment:
do we want these enabled or should we just skip the addons for this example? I haven't delved into MWAA but seems like we'd want some autoscaling to support

Reply:
for now I'm enabling them but we can always adjust the example in subsequent PRs